app.js

```diff
@@ -0,0 +1,24 @@
+
+/**
+ * Dependencies
+ */
+
+var debug = require('debug')('app');
```

Comment:
I should have probably discussed this prior. I like [`debug`](https://github.com/visionmedia/debug) for the following reasons:

- Millisecond diffs between logs helps keep an eye of performance
- Named debuggers per module mean you can turn on debugging for a single part of your app
- Output is coloured per module so you can visual group related output
- Works in browser and node

That said, I'm not opposed if you'd like me to switch back to the other solution :)